CONTRIBUTING.md

```diff
@@ -0,0 +1,856 @@
+# Bug reports
+
+Use the [issue tracker](https://github.com/lhorie/mithril.js/issues). Do check to make sure your bug hasn't already been filed. Please give the following information, where possible:
+
+1. The version of Mithril you're using, whether it's the dev version or [the version on npm](http://npm.im/mithril.js). The version on npm may not have all the latest bug fixes, so your bug might very well be fixed in the dev version.
```

Comment:
And for what it's worth, the trailing comma for objects was fixed in IE8, and arrays in IE9.